setup.py

```diff
@@ -30,7 +30,7 @@
     'scrapyd-client>=1.0.1',
     'six>=1.9.0',
     'requests~=2.22,>=2.22.0',
-    'celery~=4.2',
+    'celery>=4.1',
```

Comment:
don't we use 4.2 on hep, which also depends on hepcrawl? why not bump celery in next instead?

Reply:
On hep we use 4.2 on next we are locked on 4.1 ad 4.2 gives us `DecodeError: No JSON object could be decoded` for error thrown in worker: `signal only works in main thread` This is old thing we decided that we wont' fix it for now.

Reply:
yeah, sorry, I misread the `>=` for a `~=`.